build.gradle

```diff
@@ -540,9 +537,6 @@ subprojects {
 
     exclude testsToExclude
 
-    if (shouldUseJUnit5)
-      useJUnitPlatform()
```

Comment:
Please keep this line ...

Reply:
Sorry, I misunderstand the `shouldUseJUnit5` variable logic, so I rollback it